mobile.css

```diff
@@ -0,0 +1,292 @@
+@import url(./style.css);
+
+@media screen and (min-width: 375px) {
+  #hero {
+    background-image: url(./images/main-bg-img.png);
+    background-size: cover;
+  }
```

Comment:
- [ ] Kindly make sure the background image for the About page matches the original design:
- Your design:
   ![image](https://github.com/HtetWaiYan7191/capstone-project/assets/104968236/d2e76fa2-e3d2-4432-a53c-8abdda979858)
- Original design:
  ![image](https://github.com/HtetWaiYan7191/capstone-project/assets/104968236/c72d41fb-2779-4acd-a587-310e812eec84)
- [ ] The same applies to the main page as you're using the same selector for both. ✔